mne/decoding/__init__.py

```diff
@@ -10,4 +10,5 @@
 from .time_gen import GeneralizationAcrossTime, TimeDecoding
 from .time_frequency import TimeFrequency
 from .receptive_field import ReceptiveField
+from .time_delaying_ridge import TimeDelayingRidge
```

Comment:
yeah we expose `mne.decoding.*` objects / methods, so that one is harder to change the name of